src/Server.h

```diff
@@ -214,6 +229,18 @@ class cServer
 	int m_MaxPlayers;
 	bool m_bIsHardcore;
 
+	// TODO: std::map<int, cForgeMods &> data structure?
```

Comment:
Yeah, you should be able to do std::map<int, cForgeMods>, and then index it using the protocol number (then you wouldn't have to have a separate variable for every version, you could just create a cForgeMods instance whenever someone registers a new protocol).